main.tf

```diff
@@ -52,6 +28,12 @@ resource "aws_instance" "dp_web" {
   subnet_id                   = "${aws_subnet.data_pipe_apps.id}"
   private_ip                  = "${var.dp_web_private_ip}"
 
+  user_data = <<EOF
+  $WSPass = aws --region eu-west-2 ssm get-parameter --name wherescape_rds_user --with-decryption
+  $GPPass = aws --region eu-west-2 ssm get-parameter --name wherescape_rds_user --with-decryption
```

Comment:
same pass for both?
do you want something like
`aws --region eu-west-2 ssm get-parameter --name wherescape_rds_user --with-decryption --query 'Parameter.Value' --output text`
?